src/sentry/tasks/sentry_apps.py

```diff
@@ -121,7 +129,7 @@ def send_alert_event(event, rule, sentry_app_id):
         resource="event_alert", action="triggered", install=install, data=data
     )
 
-    send_and_save_webhook_request(sentry_app.webhook_url, sentry_app, request_data)
```

Comment:
I'm not sure if the argument change here is going to be problematic for tasks that will have been queued before the code change but then executed after. Maybe someone with a little more expertise can chime in.

cc @wedamija

Reply:
I think this is fine - we're not changing the signature of the task itself.